src/bioregistry/data/curated_mappings.sssom.tsv

```diff
@@ -1,115 +1,115 @@
+bioregistry:dso	Not	skos:exactMatch	lov:dso	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:eo	Not	skos:exactMatch	bioportal:EO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:eol	Not	skos:exactMatch	fairsharing:FAIRsharing.3J6NYn	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:epio	Not	skos:exactMatch	fairsharing:FAIRsharing.p6412v	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:epo	Not	skos:exactMatch	aberowl:EPO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:epo	Not	skos:exactMatch	bioportal:EPO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:epso	Not	skos:exactMatch	fairsharing:FAIRsharing.p6412v	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:genbase	Not	skos:exactMatch	fairsharing:FAIRsharing.3da56b	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	aberowl:GEO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	biocontext:GEO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	bioportal:GEO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	fairsharing:FAIRsharing.27rndz	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	lov:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	obofoundry:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	ols:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	ontobee:GEO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	zazuko:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geonames	Not	skos:exactMatch	fairsharing:FAIRsharing.56a0Uj	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gnd	Not	skos:exactMatch	prefixcommons:gnd	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gold	Not	skos:exactMatch	lov:gold	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gro	Not	skos:exactMatch	biocontext:GRO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gro	Not	skos:exactMatch	miriam:gro	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gro	Not	skos:exactMatch	obofoundry:gro	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:hp	Not	skos:exactMatch	aberowl:HP_O	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration		1457	2025-03-31
```

Comment:
I personally don't think adding the issue number is critical but if we are going to do this then these 3 mappings should actually refer to #1434